[BUG] conversion from 'uint8_t'

<!--

Please follow the instructions below. Failure to do so may result in your issue being closed.

### Before Reporting a Bug

1. Test with the `bugfix-2.0.x` branch to see whether the issue still exists.

2. Get troubleshooting help from the Marlin community to confirm it's a bug and not just a configuration error. Links at https://github.com/MarlinFirmware/Marlin/issues/new/choose

### Instructions

1. Fill out every section of the template below.

2. Always attach configuration files, regardless of whether you think they are involved.

3. Read and understand Marlin's Code of Conduct. By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/MarlinFirmware/Marlin/blob/master/.github/code_of_conduct.md

-->

### Bug Description

Does anyone know why i get this error when i want to activate mmu2?
I suspect that # 20985 has to do with this change?
<!-- Describe the bug in this section. (You can remove this invisible comment.) -->

### Configuration Files
[Configuration.h](https://github.com/JohnnyTheOne/MarlinJTO/blob/JTO-CoreXY/Marlin/Configuration.h)
[Configuration_adv.h](https://github.com/JohnnyTheOne/MarlinJTO/blob/JTO-CoreXY/Marlin/Configuration_adv.h)

### Steps to Reproduce

<!-- Describe the steps needed to reproduce the issue. (You can remove this invisible comment.) -->

1. [First Step] activate mmu2


**Expected behavior:**

<!-- Describe what you expected to happen here. (You can remove this invisible comment.) -->

**Actual behavior:**

```cpp
Marlin\src\feature\mmu\mmu2.cpp: In static member function 'static void MMU2::mmu_loop()':
Marlin\src\feature\mmu\mmu2.cpp:291:36: error: conversion from 'uint8_t' {aka 'unsigned char'} to 'EnsureDouble' is ambiguous
  291 |           DEBUG_ECHO_F(cmd_arg, DEC);
      |                                    ^
In file included from Marlin\src\feature\mmu\../../inc/../HAL/./STM32/../../core/serial_hook.h:25,
                 from Marlin\src\feature\mmu\../../inc/../HAL/./STM32/MarlinSerial.h:27,
                 from Marlin\src\feature\mmu\../../inc/../HAL/./STM32/HAL.h:34,
                 from Marlin\src\feature\mmu\../../inc/../HAL/HAL.h:30,
                 from Marlin\src\feature\mmu\../../inc/MarlinConfig.h:31,
                 from Marlin\src\feature\mmu\mmu2.cpp:23:
Marlin\src\feature\mmu\../../inc/../HAL/./STM32/../../core/serial_base.h:54:16: note: candidate: 'EnsureDouble::EnsureDouble(float)'
   54 |   FORCE_INLINE EnsureDouble(float a) : a(a) {}
      |                ^~~~~~~~~~~~
Marlin\src\feature\mmu\../../inc/../HAL/./STM32/../../core/serial_base.h:53:16: note: candidate: 'EnsureDouble::EnsureDouble(double)'
   53 |   FORCE_INLINE EnsureDouble(double a) : a(a) {}
      |                ^~~~~~~~~~~~
In file included from Marlin\src\feature\mmu\../../inc/MarlinConfig.h:55,
                 from Marlin\src\feature\mmu\mmu2.cpp:23:
Marlin\src\feature\mmu\../../inc/../core/serial.h:107:40: note:   initializing argument 1 of 'void SERIAL_ECHO_F(EnsureDouble, int)'
  107 | inline void SERIAL_ECHO_F(EnsureDouble x, int digit = 2) { SERIAL_IMPL.print(x, digit); }
      |                           ~~~~~~~~~~~~~^
Compiling .pio\build\BIGTREE_GTR_V1_0\src\src\gcode\bedlevel\M420.cpp.o
*** [.pio\build\BIGTREE_GTR_V1_0\src\src\feature\mmu\mmu2.cpp.o] Error 1
```

replace with `DEBUG_ECHO(cmd_arg, DEC);` get a new error

`Marlin\src\feature\mmu\mmu2.cpp: In static member function 'static void MMU2::mmu_loop()':
Marlin\src\feature\mmu\mmu2.cpp:291:34: error: no matching function for call to 'SERIAL_ECHO(uint8_t&, int)'
  291 |           DEBUG_ECHO(cmd_arg, DEC);
      |                                  ^
In file included from Marlin\src\feature\mmu\../../inc/MarlinConfig.h:55,
                 from Marlin\src\feature\mmu\mmu2.cpp:23:
Marlin\src\feature\mmu\../../inc/../core/serial.h:99:6: note: candidate: 'template<class T> void SERIAL_ECHO(T)'
   99 | void SERIAL_ECHO(T x) { SERIAL_IMPL.print(x); }
      |      ^~~~~~~~~~~
Marlin\src\feature\mmu\../../inc/../core/serial.h:99:6: note:   template argument deduction/substitution failed:
Marlin\src\feature\mmu\mmu2.cpp:291:34: note:   candidate expects 1 argument, 2 provided
  291 |           DEBUG_ECHO(cmd_arg, DEC);
      |                                  ^
In file included from Marlin\src\feature\mmu\../../inc/MarlinConfig.h:55,
                 from Marlin\src\feature\mmu\mmu2.cpp:23:
Marlin\src\feature\mmu\../../inc/../core/serial.h:103:13: note: candidate: 'void SERIAL_ECHO(serial_char_t)'
  103 | inline void SERIAL_ECHO(serial_char_t x) { SERIAL_IMPL.write(x.c); }
      |             ^~~~~~~~~~~
Marlin\src\feature\mmu\../../inc/../core/serial.h:103:13: note:   candidate expects 1 argument, 2 provided
*** [.pio\build\BIGTREE_GTR_V1_0\src\src\feature\mmu\mmu2.cpp.o] Error 1`

if i disable #define MMU2_DEBUG it works fine,
thanks

--- a/Marlin/src/feature/mmu/mmu2.cpp
+++ b/Marlin/src/feature/mmu/mmu2.cpp
@@ -287,9 +287,7 @@ void MMU2::mmu_loop() {
         else if (WITHIN(cmd, MMU_CMD_F0, MMU_CMD_F4)) {
           // filament type
           int filament = cmd - MMU_CMD_F0;
-          DEBUG_ECHOPAIR("MMU <= F", filament, " ");
-          DEBUG_ECHO(cmd_arg);
-          DEBUG_EOL();
+          DEBUG_ECHOLNPAIR("MMU <= F", filament, " ", cmd_arg);
           tx_printf_P(PSTR("F%d %d\n"), filament, cmd_arg);
           state = 3; // wait for response
         }

--- a/Marlin/src/feature/mmu/mmu2.cpp
+++ b/Marlin/src/feature/mmu/mmu2.cpp
@@ -288,7 +288,7 @@ void MMU2::mmu_loop() {
           // filament type
           int filament = cmd - MMU_CMD_F0;
           DEBUG_ECHOPAIR("MMU <= F", filament, " ");
-          DEBUG_ECHO_F(cmd_arg, DEC);
+          DEBUG_ECHO(cmd_arg);
           DEBUG_EOL();
           tx_printf_P(PSTR("F%d %d\n"), filament, cmd_arg);
           state = 3; // wait for response

